feature_reviser/transformer/custom_transformer.py

```diff
@@ -222,6 +226,29 @@ def __init__(self, ip4_devisor: float = 1e10, ip6_devisor: float = 1e48) -> None
         self.ip4_devisor = ip4_devisor
         self.ip6_devisor = ip6_devisor
 
+    def fit(self, X=None, y=None) -> "IPAddressEncoderTransformer":  # type: ignore
+        """
+        No need to fit anything.
+        """
+        return self
+
+    def transform(self, X: pd.DataFrame) -> pd.DataFrame:
+        """
+        Transforms the column containing the IP addresses to float column.
+        `-1` indicates that the value could not be parsed.
+
+        Args:
+            X (pandas.DataFrame): DataFrame to transform.
+
+        Returns:
+            pandas.DataFrame: Transformed dataframe.
+        """
+        function = functools.partial(
+            IPAddressEncoderTransformer.__to_float, self.ip4_devisor, self.ip6_devisor
```

Comment:
Typo in naming the variable: Should be 'divisor', and not 'devisor'. Replace `self.ip4_devisor` and `self.ip6_devisor` with `self.ip4_divisor` and `self.ip6_divisor` respectively.